Attempting to fix #70

gRegorLove opened the pull request:
I modified the asserted JSON in two slight ways, neither of which are related to this test: 1) The date format had the "T" and the seconds 2) Changed the "rels" to [] instead of {}. These updates, in conjunction with the parser update, should fix issue #70.

I've also added two assertions for the implied URL values, which pass.

aaronpk commented:
I don't understand what you mean about the "T" in the date. The JSON date has "2015-07-12 12:03" both before and after.

gRegorLove commented:
Oops, ignore that. I was getting phpunit errors (still) after I was pretty sure I'd fixed the parser, so as a double check I ran the HTML through unmung.com and replaced it in the test. That used mf2py as the parser and returned a different datetime result. I mistook the error as being from the original JSON.

I just reverted the multi-line JSON so the changes are easy to read.
